_data/calendar.yaml

```diff
@@ -43,3 +43,9 @@
   lab: 4
   signin: https://tinyurl.com/ocf95
   primary: True
+
+- date: 2017-10-10
+  week: 5
+  lecture: 5
+  lab: 5
+  signin: https://goo.gl/forms/oUQjqB4sDEABGb1o1
```

Comment:
I think you want to move `primary: True` from the previous week to here to make the [signin link](https://decal.ocf.berkeley.edu/signin) direct to this week instead of the last one.